pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -126,3 +126,15 @@ func checkIfHyperVVirtualSwitchExists() (bool, error) {
 func fixHyperVVirtualSwitch() (bool, error) {
 	return false, errors.New("Please override the default by adding an external virtual switch and set configuration")
 }
+
+func checkIfRunningAsNormalUserInWindows() (bool, error) {
```

Comment:
What about if a user not using `powershell` does this still work for `cmd` side?

@gbraad Can you please have a look to the logic side?